features/base/participants/constants.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * The set of possible XMPP MUC roles for conference participants.
+ */
+export const PARTICIPANT_ROLE = {
+    MODERATOR: 'moderator',
+    NONE: 'none',
+    PARTICIPANT: 'participant'
```

Comment:
@legastero, our coding style requires documentation comments on everything.

Additionally, even though it may sound to you like the enum names are as clear to everyone as to you, I don't feel that clear on what these three roles mean. For example, should I expect to never have an existing participant with role NONE? Or should I rather expect to have a participant start with role NONE, live some time with it until something happens and then the participant gets a role MODERATOR or PARTICIPANT? Does any of the previous two apply to the local participant only? Or to remote participants? You see, there's quite a lot to say documentation comments on these seemingly obvious enum names.